src/config/scrivito.ts

```diff
@@ -29,8 +29,7 @@ export function configureScrivito() {
     },
   }
 
-  if (!import.meta.env.SCRIVITO_TENANT) {
-    // Multitenancy mode
+  if (isMultitenancyEnabled()) {
     config.extensionsUrl = `/_scrivito_extensions.html?tenantId=${tenant}`
   }
```

Comment:
I think its is safe to always set `extensionsUrl` to `/_scrivito_extensions.html?tenantId=${tenant}` regardless of `isMultitenancyEnabled()`. With that, we no longer need to have a `const config`, but could call `configure()` directly.